portfolio/src/main/webapp/comments.js

```diff
@@ -110,10 +113,25 @@ function createMessageElements(comment) {
   return messageDiv;
 }
 
-function deleteComment(comment) {
+function deleteComment(comment, messageDiv) {
   const params = new URLSearchParams();
   params.append('id', comment.id);
-  fetch('/delete-comment', {method: 'POST', body: params});
+  fetch('/delete-comment', {method: 'POST', body: params})
+  .then(response => response.json())
```

Comment:
Kinda strange to have two .then statements.  Combine them into one.

Reply:
Rewrote method to use response status instead. Much neater now!

Reply:
Is there another better way to do this? This Mozilla doc linked below uses two `.then` statements for example, and I believe the goog tutorial also did it this way?   

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch